src/app/core/services/utility.service.spec.ts

```
@@ -61,7 +69,7 @@ describe('UtilityService', () => {
   it('should discard null characters', () => {
     const mockStr = 'Fyle\u0000 Expense!\u0000';
     expect(utilityService.discardNullChar(mockStr)).toEqual(
-      mockStr.replace(/[\u0000][\u0008-\u0009][\u000A-\u000C][\u005C]/g, '')
+      mockStr.replace(/[\u0000][\u0008-\u0009][\u000A-\u000C][\u005C]/g, ''),
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)